crates/core_arch/src/x86/fma.rs

```diff
@@ -32,7 +33,7 @@ use stdarch_test::assert_instr;
 #[cfg_attr(test, assert_instr(vfmadd))]
 #[stable(feature = "simd_x86", since = "1.27.0")]
 pub unsafe fn _mm_fmadd_pd(a: __m128d, b: __m128d, c: __m128d) -> __m128d {
-    vfmaddpd(a, b, c)
+    simd_fma(a, b, c)
```

Comment:
It just lowers to `llvm.fma.v*` with the llvm codegen backend. According to https://llvm.org/docs/LangRef.html#llvm-fma-intrinsic, it returns the same result as the corresponding libm `fma` function, which I think means that for vectors it will return the same result as applying `fma` to every lane.

Reply:
Thanks, that sounds correct.